style.css

```diff
@@ -55,28 +66,59 @@
   line-height: 20px;
 }
 
+.head .open {
+  display: block;
+  background: #e5e5e5;
+  height: 755px;
+  border-bottom-left-radius: 35px;
+  border-bottom-right-radius: 35px;
+  z-index: 5;
+}
```

Comment:
- [ ] Kindly give the mobile menu `height: 100vh` so it takes the whole height of the page and looks like the Figma design.

Reply:
Okay, thanks for the feedback @ShahierNashaat!